test/function/top-level-side-effects-are-preserved/_config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-	description: 'top level side effects are preserved'
+	description: 'top level side effects on imports are preserved'
```

Comment:
I think this change was an accidental one by me and should be left out. I think I force-pushed my branch to remove it.